js/buildAll.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+from os.path import abspath, dirname
+
+build_script = "build.sh"
+if "win" in sys.platform:
```

Comment:
use `sys.platform.startswith("win")`

do you know that macOS platform name is "darwin" ?